README.md

```diff
@@ -1 +1,79 @@
-# metric_webapp
\ No newline at end of file
+# Metric_webapp: Covid 19 update imformation
+
+
+This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that you will create making use of React and Redux.
+
+We will select an API that provides numeric data about a topic that you like and then build the webapp around it. The webapp will have several pages:
+
+one page with a list of items that could be filtered by some parameters; This page should be your homepage.
+one page for the item details;
+
+## Built With
+
+- React-Redux
+- Style-components
+- CSS3
+- Jest
+
+## Technologies used
+
+- Github
+- Gitflow
+- Visual Studio Code
+
+## Getting Started
+
+To clone this project :
+
+Run command: git clone `https://github.com/Somdotta07/react-redux-capstone`
+Or press the green Code button in the upper right corner and choose to download from the Download ZIP link.
+
+### Setup
+
+Add linters from linters-config
+
+Set up a new project Create React App (CRA)
+
+to create a React application is using Create React App (CRA) and following the instructions in this guide
+
+```
+- npx create-react-app react-redux-capstone
+- cd react-redux-capstone
+- npm start
+```
+
+### Add react-redux
+
+`npm install react-redux`
+
+Add React Router
+
+`npm install react-router-dom@6`
+
+## Author 
+
+😊 Bandonkeye Arnaud
+
+- Github: [ArnaudBand](https://github.com/ArnaudBand)
+- LinkeDIn: [Arnaud](https://www.linkedin.com/in/arnaudbandonkeye/)
+- Twitter: [Arnaud](https://www.twitter.com/@ba104781)
+
+### 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+### Show your support
+
+Give a ⭐️ if you like this project!
+
+### Acknowledgments
+
+Hat tip to anyone whose code was used
+Inspiration
+etc
+
+### 📝 License
+
+This project is MIT licensed.
```

Comment:
- [ ] Kindly make the MIT licensed. a clickable link as found in the Microverse README template. This will render your README professional.